.changes/unreleased/Features-20220404-190439.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Features
+body: Add a variable called selected_resources in the Jinja context containing a list
```

Comment:
@gshank could there be situations where `selected_resources` is referenced before it got set by `get_graph_queue `? I think there isn't but want to double-check.

Reply:
I think the selected_resources will be empty unless it's execution time, which should be okay.  I suppose the list could be really big for a run command that executes the whole project, but I think that should be okay too; it doesn't do much except reference the list of nodes.